frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -12,14 +11,11 @@ export const getCurrentFiscal = (fiscals: IFiscal[], key: FiscalKeys) => {
   return _.find(fiscals, { fiscalYear: currentFiscal, key: key });
 };
 
-export const getMostRecentEvaluation = (evaluations: IEvaluation[], key: EvaluationKeys) => {
-  const currentFiscal = getCurrentFiscalYear();
-
+export const getMostRecentEvaluation = (
+  evaluations: IEvaluation[],
+  key: EvaluationKeys,
+): IEvaluation | undefined => {
   const mostRecentEvaluation = _.find(_.orderBy(evaluations, 'date', 'desc'), { key: key });
-  const fiscal = getFiscalYear(mostRecentEvaluation?.date);
```

Comment:
simplified this further, now just ensuring that all dates sent to the API are using the current fiscal year.